installer/pkg/gcp/broker.go

```diff
@@ -48,10 +48,18 @@ func EnableAPIs(apis []string) error {
 
 // enabledAPIs returned set of enabled GCP APIs.
 func enabledAPIs() (map[string]bool, error) {
+	var output []byte
+	var err error
+
 	cmd := exec.Command("gcloud", "service-management", "list", "--format", "json")
-	output, err := cmd.Output()
+	output, err = cmd.Output()
 	if err != nil {
-		return nil, fmt.Errorf("failed to retrived enabled GCP APIs : %v", err)
+		// In the latest version of `gcloud`, "service-management" command group becomes "services".
```

Comment:
This simple check can lead to false positives.

I'd recommend to wrap all calls to the service-management group into separate function.
That function can determine which command group it needs to use, by checking more reliably for the error.
Specifically, the promising error message looks like: "Invalid choice: 'service-management'"
And, it can also cache the command group name: "" (undetermined), "service-management", "services".

Reply:
I am not sure if I understood your proposal, so instead of wrapping all the calls to Service Management API , I extracted out the logic to pick command group name. Users won't know anything about which command group sc installer used.